### PR TITLE
Only mess with sdp if h264 is supported

### PR DESCRIPTION
**Pull request by brianpeiris**
If the client doesn't support h264 (e.g. WebARonARCore, chromium), avoid setting sdp to a particular h264 profile.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "webpack-merge": "^4.1.0"
   },
   "dependencies": {
+    "codec-detect": "^1.3.0",
```
Comment (brianpeiris) [inline package.json]: Published from my fork of the original library: https://github.com/brianpeiris/codec-detect

**Comment by robertlong** [comment]
Seems like a good idea to add feature detection. I assume this breaks things on a few browsers now? 

Also, can we just use a helper method like this?

```
function isH264Supported() {
  const video = document.createElement("video");
  return video.canPlayType('video/mp4; codecs="avc1.42E01E, mp4a.40.2"') !== "";
}
```
I don't think we need that whole library.

**Comment by brianpeiris** [comment]
Thanks!

**Comment by brianpeiris** [comment]
I'll add `np` to the project and publish a new release.

**Pull request merged**